### DISCUSSION TITLE
Table from two dimensional array

### DISCUSSION THREAD

**Discussion by raedma** [jtablesaw/tablesaw#1042] [Q&A]
Is there a convenient possibility to create a `Table` by means of a two-dimensional array of values and a one dimensional array of column names?

E.g.

    double[][] data = new double[][]{{0.1,0.2,0.3},{0.2,0.3,0.4},{0.1,0.3,0.4}};
    String[] columnNames = new String[]{"x","y","z"};
    
    Table.create("Test)
            .fromColumns(data,columnNames);

What is the most convenient way to this currently? I would suspect there is a more elegant way than to extract the columns from the array?

**Comment by lwhite1** [comment]
I don't think so, unfortunately. There are utilities for going the other
way in the conversion package.  This would be a good addition.
